src/types/schema.ts

```diff
@@ -45,6 +45,23 @@ export type ValueSchemaVariant = {
   variants: Variant[];
 } & ValueSchemaMetadata;
 
+export type ValueSchemaOption = {
+  display: string;
+  value: string;
+};
+
+export type UIValueSchemaWithOptionsSingle = {
+  optionLabel: string;
```

Comment:
per discussion with @duranb we decided to just call this `label` to reduce confusion